### ISSUE TITLE
41 vulnerabilities (34 moderate, 7 high)

### ISSUE THREAD

**Issue by nezaboravi** [JeffreyWay/laravel-mix#2999]
-  Laravel Framework 8.46.0
-   Laravel Mix Version: npm list --depth=0

├── laravel-mix@6.0.19
├── lodash@4.17.21
└── postcss@8.3.0

-   Node Version (`node -v`): v15.10.0
-   NPM Version (`npm -v`): 7.6.0
-   OS: MacOSx Big Sur version 11.4, Apple silicon

### Description:
Just installed fresh Laravel, and run npm install
I am getting this error: 
41 vulnerabilities (34 moderate, 7 high)

![image](https://user-images.githubusercontent.com/1515172/121338313-adad9180-c91d-11eb-8f84-d31eafcc44d8.png)

### Steps To Reproduce:
Install new laravel instance
Run npm install

**Comment by dransidu** [comment]
Same here how to fix this?

**Comment by TeachHip** [comment]
Same here

**Comment by jasonloeve** [comment]
Same, waiting on an update release.

<img width="579" alt="Screen Shot 2021-06-11 at 8 24 53 AM" src="https://user-images.githubusercontent.com/1393754/121592267-9215c880-ca8e-11eb-87a7-c231997109f9.png">

**Comment by marczh** [comment]
Same Problem here.

node -v = v16.3.0
npm -v = 7.17.0

tried this:
├── axios@0.21.1
├── laravel-mix@6.0.6
├── lodash@4.17.19
└── postcss@8.1.14

and tried this:
├── axios@0.21.1
├── laravel-mix@6.0.19
├── lodash@4.17.21
└── postcss@8.3.4

both not working

**Comment by thecrypticace** [comment]
So I updated some stuff in our `master` branch over the weekend however, even if we tag a release, it won't fix some vulnerabilities still present in dependencies.

1. One is present in css-what depended upon by svgo.
2. One is present in glob-parent due to `@types/browser-sync` pulling in an old version of chokidar.
3. One is because `resolve-url-loader` pulls in a vulnerable version of postcss (when it should be a peer dep).

We should still tag a new release though. I'll see if we can do that soon.

**Comment by marczh** [comment]
With laravel-mix@6.0.20 on a fresh laravel install now I get less vulnerabilities.

10 vulnerabilities (4 moderate, 6 high)

[npm-audit-log.txt](https://github.com/JeffreyWay/laravel-mix/files/6670018/npm-audit-log.txt)

**Comment by johnneijzen** [comment]
so far my side only 7 high vulnerabilities from css-what so it is getting better

![image](https://user-images.githubusercontent.com/1343960/122701303-7f19aa00-d27f-11eb-81a9-33378eeaf569.png)

**Comment by johnneijzen** [comment]
https://github.com/svg/svgo/pull/1485 i guess this only update left that we are waiting

**Comment by johnneijzen** [comment]
This issue should be fixed now svgo release update two days ago 

![image](https://user-images.githubusercontent.com/1343960/123559231-5ccae380-d7cd-11eb-95f0-4a598191f379.png)

**Comment by thecrypticace** [comment]
Confirmed. I'll note that I _had_ to run an `npm upgrade` even from a clean install without a lock file. yarn didn't have this problem. idk why npm is being like that but something to note.

Aa this has been addressed I'm closing the issue.
